lib/drizzlepac/tweakutils.py

```diff
@@ -783,7 +783,15 @@ def make_vector_plot(coordfile,columns=[1,2,3,4],data=None,figure_id=None,
             Write out plot to a file with this name if specified.
 
     """
+    import matplotlib
     from matplotlib import pyplot as plt
+    interactive = matplotlib.is_interactive()
+    turn_interactive_off = False
+
+    if not interactive:
+        plt.ion()
+        if not interactive:
```

Comment:
Is this second `if` even necessary? It seems that the first one already selects a branch that corresponds to "non-interactive" mode.

Reply:
The difference between interactive and non-interactive mode is when the plots get updated to the screen. I'm unsure how the plotting functions are being used specifically in this package. If the intention is to generate a bunch of plots that stay on the screen for the user to look at before continuing, then non-interactive mode is commonly used. Using the show() function will throw the plots up together and block until you want to destroy them. But if interactive is already on for some other reason,  you can still do plotting without switching between the two.

Reply:
The difference between interactive and non-interactive mode is when the plots get updated to the screen. I'm unsure how the plotting functions are being used specifically in this package. If the intention is to generate a bunch of plots that stay on the screen for the user to look at before continuing, then non-interactive mode is commonly used. Using the show() function will throw the plots up together and block until you want to destroy them. But if interactive is already on for some other reason, you can still do plotting without switching between the two.